.gitignore

```diff
@@ -3,4 +3,6 @@ dist
 
 node_modules
 *-error.log
-*-debug.log
\ No newline at end of file
+*-debug.log
+
+react-app-env.d.ts
```

Comment:
Is this still needed since we ditched react-scripts?

Reply:
Good call! This file won't be generated anymore, so I will remove .gitignore